db/migrate/20180824144803_change_name_in_hotels.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeNameInHotels < ActiveRecord::Migration[5.2]
+  def up
+    change_column :hotels, :name, :string, null: false
+    add_column :hotels, :start_time, :datetime, null: false
```

Comment:
You're adding columns here, but not removing them in `down` method - another reason for using `change` :D